_docs-v3/event-display/eventAfterRender.md

```diff
@@ -8,3 +8,11 @@ Triggered after an event has been placed on the calendar in its final position.
 <div class='spec' markdown='1'>
 function( *event*, *element*, *view* ) { }
 </div>
+
+`event` is the [Event Object](event-object) that is attempting to be rendered.
+
+`element` is a newly created jQuery element that will be used for rendering. It has already been populated with the correct time/title text.
```

Comment:
one minor modification actually... at the point where eventAfterRender is called, `element` has already been "rendered" (aka, placed into the DOM and positioned)... it's not that it "will be used for rendering" ... rendering already happened. Instead, I would emphasize that the element is already in the DOM and its coordinates have been set.